test/e2e/clusterctl_upgrade_test.go

```diff
@@ -25,7 +25,8 @@ import (
 	"sigs.k8s.io/cluster-api/test/framework"
 )
 
-var _ = Describe("When testing clusterctl upgrades (v0.3=>current)", func() {
+// Note: v1.4 is the last version which serves v0.3 APIs. This is the last CAPI version that v0.3 types can be upgraded to.
+var _ = Describe("When testing clusterctl upgrades (v0.3=>v1.4) [PR-Informing]", func() {
```

Comment:
If I'm not mistaken we can just delete this test.

It is installing 0.3 and upgrading to 1.4. All based on GitHub releases. (with the exception of cluster templates)

We already have a test from v0.3 to 1.4 on the release-1.4 branch

Reply:
Sure - we can also get rid of the templates, right?